src/SWP/Bundle/CoreBundle/Theme/TenantAwareThemeContext.php

```diff
@@ -49,11 +49,11 @@
     /**
      * TenantAwareThemeContext constructor.
      *
-     * @param TenantContextInterface   $tenantContext   Tenant context
-     * @param ThemeRepositoryInterface $themeRepository Theme repository
-     * @param CacheProvider            $cacheService    Cache Service
+     * @param TenantContextInterface             $tenantContext   Tenant context
+     * @param ReloadableThemeRepositoryInterface $themeRepository Theme repository
+     * @param CacheProvider                      $cacheService    Cache Service
      */
-    public function __construct(TenantContextInterface $tenantContext, ThemeRepositoryInterface $themeRepository, CacheProvider $cacheService)
+    public function __construct(TenantContextInterface $tenantContext, ReloadableThemeRepositoryInterface $themeRepository, CacheProvider $cacheService)
```

Comment:
`reloadThemes` method is not used here so `ThemeRepositoryInterface` typing can stay here.